editor/osm_auth.cpp

```diff
@@ -6,20 +6,16 @@
 
 #include "base/string_utils.hpp"
 
-#include "private.h"
+#include "cppjansson/cppjansson.hpp"
 
-#include "3party/liboauthcpp/include/liboauthcpp/liboauthcpp.h"
```

Comment:
Can this 3party library be completely removed?

Reply:
We don't use `liboauthcpp` lib for OAuth2. In OM it's used to calculate SHA1.
I think we can remove it in PR #7658